src/Integration.Vsix.UnitTests/SonarLintTagger/TaggerProviderTests.cs

```diff
@@ -210,10 +264,15 @@ public void CreateTagger_should_return_new_tagger_for_already_tracked_file()
         }
 
         [TestMethod]
-        public void CreateTagger_close_last_tagger_should_unregister_tracker()
+        public void CreateTagger_CloseLastTagger_ShouldUnregisterTrackerAndRaiseEvent()
         {
             var doc1 = CreateMockedDocument("doc1.js");
 
+            object actualSender = null;
```

Comment:
Updated the existing test to check the event is only raised when the last tagger is removed.